code/game/objects/items/plushes.dm

```diff
@@ -9,11 +9,52 @@
 	var/list/squeak_override //Weighted list; If you want your plush to have different squeak sounds use this
 	var/stuffed = TRUE //If the plushie has stuffing in it
 	var/obj/item/grenade/grenade //You can remove the stuffing from a plushie and add a grenade to it for *nefarious uses*
+	//--love ~<3--
+	var/male_chance = 1
```

Comment:
this should be a single var, gender, where 1 is male, 0 is female and 2 is randomly choose on INITIZIALIZE the gender.

use defines PUSH_MALE, PLUSH_FEMALE, PLUSH_RANDOM

Reply:
>binary genders

smh

Reply:
I will do that. I'm not sure my PR's scope requires specific gender weighting until there is a catgrille plush to simulate bagil population.

Reply:
Just use the built in BYOND defines for gender and have NEUTRAL be random

Reply:
yeah okay that will work fine as well